### ISSUE TITLE
Error with running App with react-native-ble on Android in debug mode

### ISSUE THREAD

**Issue by mosesmc52** [jacobrosenthal/react-native-ble#37]
Hi, 

I'm attempting to run App on react-native-ble on my Adroid, but I recieve this error. How would I fix it?

AndroidManifest.xml:9:5-11:41 Error:
uses-sdk:minSdkVersion 16 cannot be smaller than version 21 declared in library

Suggestion: use tools:overrideLibrary="com.geniem.rnble" to force usage

**Comment by warrenronsiek** [comment]
1) Does this only happen when you run in debug mode?
2) Assuming you aren't using an emulator (which wont work), it sounds like your phone's OS is outdated. Have you tried updating it?

**Comment by mosesmc52** [comment]
Yes, I'm not using the debug mode. I believe the reason is caused because my phone OS is outdated.

**Comment by mosesmc52** [comment]
So I purchased (this)[https://www.amazon.com/gp/product/B01N1KIKSA/ref=oh_aui_detailpage_o03_s00?ie=UTF8&psc=1] tablet 

The OS it's using is Android 6.0. I also downgraded to React Native v0.38.0. After linking react-native-ble,  I get the same error:

AndroidManifest.xml:9:5-11:41 Error:
uses-sdk:minSdkVersion 16 cannot be smaller than version 21 declared in library

I can't image that the OS is out of date.
